DEBUG 2019-09-30 21:36:51.155 UTC: 	 Executing command: 'submit'
DEBUG 2019-09-30 21:36:51.155 UTC: 	 Found CRAB cache file /afs/cern.ch/user/s/schhibra/.crab3
DEBUG 2019-09-30 21:36:51.164 UTC: 	 Checking credentials
DEBUG 2019-09-30 21:36:51.166 UTC: 	 Found proxy file /tmp/x509up_u59012
DEBUG 2019-09-30 21:36:51.166 UTC: 	 Getting proxy life time left
DEBUG 2019-09-30 21:36:52.173 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u59012 -timeleft
 output : 84143

 error: 
 retcode : 0
DEBUG 2019-09-30 21:36:52.981 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u59012 -actimeleft
 output : 84143

 error: 
 retcode : 0
DEBUG 2019-09-30 21:36:52.981 UTC: 	 Proxy valid for 23:22:23 hours
INFO 2019-09-30 21:36:52.981 UTC: 	 Will use CRAB configuration file crabConfig.py
DEBUG 2019-09-30 21:36:52.981 UTC: 	 Loading CRAB configuration file.
ERROR 2019-09-30 21:36:53.017 UTC: 	 Error while loading CRAB configuration:
'DYJetsToLL_M-10to50_L1_12_5_L1_15_7'
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
ERROR 2019-09-30 21:36:53.018 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1909/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1909/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1909/lib/python2.7/site-packages/CRABClient/Commands/submit.py", line 34, in __init__
    SubCommand.__init__(self, logger, cmdargs, disable_interspersed_args=True)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1909/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 311, in __init__
    self.loadConfig(self.options.config, self.args)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.1909/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 87, in loadConfig
    raise ConfigurationException(configmsg)
ConfigurationException: Error while loading CRAB configuration:
'DYJetsToLL_M-10to50_L1_12_5_L1_15_7'
Please refer to https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3CommonErrors#Syntax_error_in_CRAB_configurati
See the ./crab.log file for more details.
The documentation about the CRAB configuration file can be found in https://twiki.cern.ch/twiki/bin/view/CMSPublic/CRAB3ConfigurationFile
